pkg/vpcmodel/explainabilityPrint.go

```diff
@@ -446,3 +471,21 @@ func getLayersToPrint(filtersRelevant map[string]bool, isIngress bool) (filterLa
 	}
 	return orderedRelevantFiltersLayers
 }
+
+func (d *detailedConn) respondString() string {
+	switch {
+	case d.allConn.Equal(d.nonTCP):
+		// no tcp component - ill-relevant
+		return ""
+	case d.tcpRspEnable.IsEmpty():
+		// no tcp responsive component
+		return "\n\tTCP respond is blocked"
+	case d.tcpRspEnable.Equal(d.allConn):
+		// tcp responsive component is the entire connection
+		return "\n\tThe entire connection is TCP responsive"
+	case d.tcpRspDisable.IsEmpty():
+		return "\n\tThe TCP sub-connection is responsive"
+	default:
+		return "\n\tTCP respond is enabled on " + d.tcpRspEnable.String()
+	}
```

Comment:
Hi
I have a general comment, "who is the receiver" comment, it has two points:
1. A good practice is to put all the function of a receiver in one file.
2. the receiver is not just another parameter, the main context of the function should be the receiver itself
3. if you still think it is better not to put a func in another file, at list put all the functions of the receiver together.

in this function, I understand why you do not want to put it in the function of detailedConn, the context of this function is explainability, not detailedConn. I dont think that detailedConn should be the receiver here, just a parameter.

Reply:
Good comment. However, the changes implied by this comment are very likely to involve general refactoring - mainly moving functionality around. I suggest that in this PR we will restrict the changes just to decide for the new functions yes/no to have the receiver and in a separate PR (probably for V0.5) we will handle the general comment across the board.

Reply:
Modified to be a parm and not the receiver

Reply:
Yes, I do not think that moving code during PR review is a good idea.
however, I didnt dive into it, but I think that in most placers I comment, the receiver should not be the reciever.
please check

Reply:
Please open an issue and we will discuss it. I used explain structs as receivers for explain functionality across the board.